spec/consensus/gossip/crdt.md

```diff
@@ -68,59 +62,64 @@ To save memory, entries should be removed from the tuple space as soon as they b
 For example, if a new height is started, all entries corresponding to previous heights become stale.
 
 In general, simply forgetting stale entries in the local view would save the most space.
-However, it could lead to entries being added back and never being completely purged from the system.
-Although stale entries do not affect the algorithm, or they would not be considered stale, not adding the entries back for performance and resource utilization sake.
+However, if the second approach described [above](#nodes-state-as-a-tuple-space), it could lead to entries being added back and never being completely purged from the system.
```

Comment:
I am not sure to understand why if a new height is started, all the entries corresponding to previous heights become stale. We need to have a decision at that height.

Reply:
You are right, the decision cannot be forgotten, but because we can get previous decisions out-of-band (from the tuple space point of view), we can forget them here.

Reply:
Good point. Although it is an implementation detail of `getValue` and `valid`, it might be worth capturing it here. I will consider it in specifying the superseding function, in the follow up PR.